core/controllers/get_filters.py

```diff
@@ -66,19 +66,27 @@ def get_outcome_categories():
     return outcome_categories, 200
 
 
-def get_regions():
-    """Returns all unique itl region codes associated with projects.
+def get_geospatial_regions():
+    """Returns all unique ITL1 region codes associated with projects.
 
-    :return: a list of itl region codes
+    :return: a list of ITL1 region codes and region names in dictionaries
     """
-    projects = Project.query.all()
-
-    itl_regions = set(region for project in projects for region in project.itl_regions)
+    geospatial_itl1_regions = (
+        GeospatialDim.query.order_by(GeospatialDim.data_blob["itl1_region_name"])
+        .distinct(GeospatialDim.itl1_region_code, GeospatialDim.data_blob["itl1_region_name"])
```

Comment:
I think we discussed this today, but just in case, I think we said, to that if we move itl_region_name to be a column rather than something hidden in the jsonb, then we can make this query much simpler / clearer (As won't need to specify which fields go in the distinct clause - it's also not very clear from this query what fields we use for what)? But I think we said to do in a separate PR / migration?

Reply:
Yep a separate PR I think just for tidyness, I'll add a ticket for this and add a TODO referencing the ticket number